layouts/partials/head.html

```diff
@@ -19,11 +19,12 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 {{ hugo.Generator }}
-{{ if eq (getenv "HUGO_ENV") "production" }}
+{{- $outputFormat := partial "outputformat.html" . -}}
+{{ if and hugo.IsProduction (ne $outputFormat "print") -}}
```

Comment:
I think we should continue to check `(getenv "HUGO_ENV") "production"`

Reply:
👀

Reply:
> @shuuji3 we need to make sure that Netlify previews of web pages (whether for print or not) _don't_ get crawled.

That's right. Maybe I should change like this:

```suggestion
{{ if and (eq (getenv "HUGO_ENV") "production") (ne $outputFormat "print") -}}
```

I assume that `HUGO_ENV` != "production" on Netlify preview and `HUGO_ENV` == "production" on the production (https://kubernetes.io). Is that right?

Reply:
I've updated the PR with the above code:`{{ if and (eq (getenv "HUGO_ENV") "production") (ne $outputFormat "print") -}}`

Note: environment variable `HUGO_ENV = "production"` on the `production` environment is specified here:
[website/netlify.toml at 971412c3fdf918606c4f40e84b493194f821fe93 · kubernetes/website](https://github.com/kubernetes/website/blob/971412c3fdf918606c4f40e84b493194f821fe93/netlify.toml#L16)

Now the website should have property as follows:

<table>
<tr>
	<th>Environment
	<th><code>HUGO_ENV
	<th>Page type
	<th><code>content</code> property of <code>&lt;meta name="ROBOTS">
<tr>
	<td>Netlify preview
	<td>-
    <td>normal page
	<td><code>NOINDEX, NOFOLLOW
<tr>
	<td>Netlify preview
	<td>-
    <td>print mode
	<td><code>NOINDEX, NOFOLLOW
<tr>
	<td>Netlify production (maybe cannot test)
	<td><code>production
    <td>normal page
	<td><code>INDEX, FOLLOW
<tr>
	<td>Netlify production (maybe cannot test)
	<td><code>production
    <td>print mode
	<td><code>NOINDEX, NOFOLLOW
<tr>
	<td>local
	<td>-
    <td>normal page
	<td><code>NOINDEX, NOFOLLOW
<tr>
	<td>local
	<td>-
    <td>print mode
	<td><code>NOINDEX, NOFOLLOW
<tr>
	<td>local
	<td><code>production
	<td>normal page
    <td><code>INDEX, FOLLOW
<tr>
	<td>local
	<td><code>production
	<td>print mode
    <td><code>NOINDEX, NOFOLLOW
</table>

On the local environment, we can switch the `HUGO_ENV` by running `export HUGO_ENV=production` command.